sdk/python/feast/infra/key_encoding_utils.py

```diff
@@ -23,6 +23,22 @@ def _serialize_val(
         raise ValueError(f"Value type not supported for Firestore: {v}")
 
 
+def _deserialize_value(value_type, value_bytes) -> ValueProto:
```

Comment:
Can you add an explicit test of this function?

Reply:
Sure. Will add one for it